licenses/statsmodels.txt

```diff
@@ -0,0 +1,34 @@
+Copyright (C) 2006, Jonathan E. Taylor
```

Comment:
After the remodeling, I wonder if including statsmodels' license file is still necessary, since the documentation text is now quite different?